### DISCUSSION TITLE
Keycloak 20 - reverse proxy --proxy=edge

### DISCUSSION THREAD

**Discussion by Satishshelake** [keycloak/keycloak#20098] [Archive]
Unable to fetch login page after enable --proxy=edge configuration in keycloak.conf file. 
It gives spinning UI when try to do login.
Does need more configuration for enable proxy?

my configuration is - 

http-enabled=true
http-port=8080
http-host=wpfhost
hostname-strict-https=false
hostname-strict-http=false
proxy=edge
http-relative-path=/auth
spi-hostname-default-strict-http=false

I am trying to use url http://wpfhost:8080/auth/admin/master/console.

**Comment by vmuzikar** [comment]
@Satishshelake Hello, if you set `--proxy` option, it expects the proxy uses the default ports, i.e. 80 and 443. If your proxy uses a different port, you need to set `--hostname-port`.

**Reply by Satishshelake** [reply]
Thank You!! It's working now.
